simplq/src/components/pages/Admin/AddMember.jsx

```diff
@@ -1,8 +1,8 @@
 import React from 'react';
 import AddIcon from '@material-ui/icons/Add';
+import JoinQueueForm from 'components/pages/Join/JoinForm';
 import SidePanelItem from 'components/common/SidePanel/SidePanelItem';
 import styles from './admin.module.scss';
-import JoinQueueForm from '../Join/Form';
```

Comment:
Can we move JoinForm and it's styling into common folder?

Reply:
Yes. I just try to avoid moving to much things at once while someone else might work on same files.